tests/tools.py

```diff
@@ -28,7 +28,7 @@
 
 import time
 
-import azurelinuxagent.common.event as event
```

Comment:
This caused a bug.  We import `as event` and then pollute all of the test files that import this file, which is basically all of them.  We have a naked event import, which then collides with other files, and we see failures in TravisCI as a result. The fix is not to import `as event`, and only import the relevant files.

Reply:
I spoke too soon.  There's a circular dependency.  I am trying to see if I need to unwind it now, or if I can work around it.  It manifests when running particular test cases.

Reply:
I moved the import to exactly where it is needed so it isn't sucked in unnecessarily. This needs a better fix.  I'll open an issue.